contextual-identities/README.md

```diff
@@ -0,0 +1,11 @@
+# Contextual Identities
+
+## What it does
+
+Lists existing idenities, lets you create new tabs with an identity and remove all tabs from an identity.
+
+## What it shows
+
+How to use the contextualIdentities API. Please note: you must have contextualIdentities enabled. You can do that by going to about:config, or if you are using web-ext by running:
```

Comment:
"You can do that by going to about:config and setting the \`privacy.userContext.enabled\` preference to true"